pkg/broker/ingress/multi_topic_decouple_sink_test.go

```diff
@@ -220,6 +226,219 @@ func TestMultiTopicDecoupleSink(t *testing.T) {
 	}
 }
 
+func TestHasTrigger(t *testing.T) {
+	DecoupleQueue := &config.Queue{Topic: "test_topic", State: config.State_READY}
+
+	tests := []struct {
+		name          string
+		brokerTargets map[string]*config.Target
+		hasTrigger    bool
+	}{
+		{
+			name:       "broker with no target",
+			hasTrigger: false,
+		},
```

Comment:
You might want to add a similar test but with an empty map (not nil).

Reply:
Done